### PR TITLE
contracts-bedrock: cleanup imports, fix storage layout

### PR DESCRIPTION
**Pull request by tynes**
**Description**

We don't need to use the upgradable initializable in the `OptimismPortal`, also fix the sizes of the storage layout spacers so that they are in sets of 50

### CODE REVIEW

File: packages/contracts-bedrock/contracts/L1/OptimismPortal.sol [hunk 1]
```diff
@@ -63,6 +63,11 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
      */
     mapping(bytes32 => bool) public finalizedWithdrawals;
 
+    /**
+     * @notice Reserve extra slots (to a total of 50) in the storage layout for future upgrades.
+     */
+    uint256[48] private __gap;
```
Comment (maurelian) [inline packages/contracts-bedrock/contracts/L1/OptimismPortal.sol:69]: I previously took this out, I don't think we need them in our concrete contracts which are not inherited by anything in production. If we put this one here, then we should have one in the `L2OutputOracle` too.

It's fine either way, but a bit cleaner not to IMO.

Comment (tynes) [reply packages/contracts-bedrock/contracts/L1/OptimismPortal.sol:69]: Ok i'm down to remove it

Comment (tynes) [reply packages/contracts-bedrock/contracts/L1/OptimismPortal.sol]: I've addressed your comment @maurelian

**Comment by tynes** [comment]
Looking for review @maurelian @smartcontracts

**Review by maurelian — APPROVED** [review]

**Comment by tynes** [comment]
This needs a rebase

**Comment by tynes** [comment]
I've rebased this, can i get a review @smartcontracts ?

**Review by smartcontracts — APPROVED** [review]

**Pull request merged**